router/auth.go

```diff
@@ -239,6 +240,22 @@ func authInjectYellowfin(log *log.Logger, w http.ResponseWriter, req *http.Reque
 		defer logoutResp.Body.Close()
 
 		return false // do not forward this to YF
+
+	// The front-end with an open Iframe to YF will keep polling to check if it is
+	// the session currently allowed to access YF. If another instance is opened with the same user,
+	// that instance will get access, and the first Iframe should exit and go to IMQS home page.
+	case "/yellowfin/checksession":
+		target.lock.RLock()
+		token_yf, exists := target.tokenMap[authData.Identity].(*yellowfinToken)
+		if !exists || token_yf.UserSession == "" {
+			fmt.Fprintf(w, "%s", "No Session Found")
```

Comment:
Would it not be better to send a response code other than 200 here, instead of making the front-end look for the magic string "No Session Found"?